core/shared/src/main/scala/scalaz/zio/Managed.scala

```diff
@@ -33,14 +33,29 @@ sealed abstract class Managed[+E, +R] { self =>
 
   final def seq[E1 >: E, R1](ff: Managed[E1, R1]): Managed[E1, (R, R1)] =
     seqWith(ff)((_, _))
+
+  final def parWith[E1 >: E, R1, R2](that: Managed[E1, R1])(f0: (R, R1) => R2): Managed[E1, R2] =
+    new Managed[E1, R2] {
+      override def use[E2 >: E1, A](f: R2 => IO[E2, A]): IO[E2, A] =
+        self.use { r =>
+          that.use { r1 =>
```

Comment:
Thanks @mijicd. This would work, but it is identical to `flatMap`; the resources would be acquired sequentially, rather than in parallel.

I don't recall what's the exact direction we discussed here, but maybe refer back to the original PR - I think there was an idea suggested in the conversation there about how to implement this combinator.